scss/components/dropdown.scss

```diff
@@ -23,6 +23,7 @@ $block: ns(dropdown);
     $fd-dropdown-menu-item-background-color: $fd-dropdown-menu-background-color !default;
     $fd-dropdown-menu-item-background-color--hover: darken(fd-color(action), 5%) !default;
     $fd-dropdown-menu-item-padding: fd-space(3) fd-space(4) !default;
+    $fd-dropdown-menu-item-disabled-color: fd-color(text, 2);
```

Comment:
Should be `$fd-dropdown-menu-item-color--disabled` to match the syntax of the other vars.